helpers/networker.py

```diff
@@ -0,0 +1,85 @@
+from helpers import remote, sysnet, tf_cfg
+
```

Comment:
Please add a header comment describing that the module does and define the static variables with licensing and so on as it is in other modules

Reply:
Fixed